### PR TITLE
add spark helper

### PR DESCRIPTION
**Pull request by Tianhao-Gu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,23 +63,37 @@ When running Spark in the Jupyter notebook container, the default `spark.driver.
 the hostname (`SPARK_DRIVER_HOST`) of the container. 
 In addition, the environment variable `SPARK_MASTER_URL` should also be configured.
 
+#### Using Predefined SparkSession from `spark.utils.get_spark_session` method
+```python
+from spark.utils import get_spark_session
```
Comment (MrCreosote) [inline README.md:68]: Will `spark` not conflict with other package names?

File: src/__init__.py
Comment (MrCreosote) [inline src/__init__.py:1]: I don't think `__init__` files are necessary any more

File: README.md (hunk 1 above)
Comment (Tianhao-Gu) [reply README.md]: I don't think so. But just in case changed the repo name to `common`.

File: src/__init__.py
Comment (Tianhao-Gu) [reply src/__init__.py:1]: 👍 removed.

File: README.md (hunk 1 above)
Comment (MrCreosote) [reply README.md]: https://pypi.org/project/common/ seems dead.

That being said, https://pypi.org/project/spark/ seems super dead

Comment (MrCreosote) [reply README.md]: So spark seems safe and is probably a better name, sorry about that

Comment (Tianhao-Gu) [reply README.md:68]: 👍

**Pull request merged**